group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -1131,32 +1132,32 @@ private <IN, OUT> OUT handleOperationException(
             case UNKNOWN_SERVER_ERROR:
                 log.error("Operation {} with {} hit an unexpected exception: {}.",
                     operationName, operationInput, exception.getMessage(), exception);
-                return handler.apply(Errors.UNKNOWN_SERVER_ERROR, null);
+                return handler.apply(Errors.UNKNOWN_SERVER_ERROR, Errors.UNKNOWN_SERVER_ERROR.message());
```

Comment:
if my concern about the message with fallback makes sense, I would expect that here we should use `apiError.message()` instead (and in the other error cases too)